tests/test_policy_inspect.py

```
@@ -152,19 +161,20 @@ def test_copyleft_policy_markdown(self):
     """
        Inspect for undeclared components empty path
     """
+
     def test_copyleft_policy_empty_path(self):
```

Ruff (0.8.2)

Redefinition of unused `test_copyleft_policy_empty_path` from line 51

(F811)